src/libs/elektra/kdbenum.c

```diff
@@ -98,6 +98,21 @@ enum elektraKeyFlags
 	KEY_END=0		/*!< Used as a parameter terminator to keyNew() */
 };
 
+/**
+ * Copy options
+ * 
+ * @ingroup key
+ * @see keyCopy()
+ */
+enum elektraCopyFlags
+{
+  	KEY_CP_NAME   = 1<<0, /*!< Flag for copying the key name */
```

Comment:
spaces broken.

Can you check if clang-format off is still needed? Maybe the latest clang-format makes this file look nice.

Reply:
Seems to work fine with `clang-format`